README.md

```diff
@@ -119,10 +125,10 @@ Alternatively, a YAML file that specifies the values for the above parameters
 can be provided while installing the chart. For example,
 
 ```
-$ helm install --name npm -f values.yaml verdaccio/verdaccio
+$ helm install npm -f values.yaml verdaccio/verdaccio
 ```
 
-> **Tip**: You can use the default [values.yaml](charts/verdaccio/values.yaml)
+> **Tip**: You can use the default [values.yaml](values.yaml)
```

Comment:
This undos https://github.com/verdaccio/charts/pull/34

Reply:
I made some changes. Please review it